### PR TITLE
Bump preseq

### PR DESCRIPTION
**Pull request by ewels**
Updating preseq with upstream changes from #382

### CODE REVIEW

**Comment by ewels** [comment]
@nf-core-bot fix linting

**Review by maxulysse — APPROVED** [review]
CHANGELOG?

File: modules/nf-core/preseq/lcextrap/main.nf [hunk 1]
```diff
@@ -1,9 +1,9 @@
 process PRESEQ_LCEXTRAP {
     tag "$meta.id"
     label 'process_single'
-    label 'error_ignore'
+    label 'error_retry'
```
Comment (maxulysse) [inline modules/nf-core/preseq/lcextrap/main.nf:4]: I think error_ignore was a good idea actually

Comment (ewels) [reply modules/nf-core/preseq/lcextrap/main.nf:4]: Check line 24 - if retrying it uses a different flag.

**Review by edmundmiller — APPROVED** [review]
LGTM, gonna update the changelog.

**Pull request merged**